src/compile/layout/header.ts

```diff
@@ -97,6 +97,30 @@ export function getHeaderGroups(model: Model, channel: HeaderChannel): VgMarkGro
   return groups;
 }
 
+export function labelAlign(angle: number) {
```

Comment:
these functions also have to take orient into account

Reply:
I'm not sure if header support orient yet, but it doesn't please add a TODO item to the facet (header) customization issue.